conf/airframes/examples/example_t4_actuators_fw.xml

```diff
@@ -0,0 +1,328 @@
+<!--      <define name="RADIO_TH_HOLD"     value="0"/> &lt;!&ndash; Throttle hold in command laws &ndash;&gt;-->
+<!--      <define name="RADIO_FMODE"       value="0"/> &lt;!&ndash; Throttle curve select &ndash;&gt;-->
+<!--      <define name="RADIO_FBW_MODE"    value="0"/> &lt;!&ndash; Switch between AP and FBW control &ndash;&gt;-->
+<!--      <define name="RADIO_KILL_SWITCH" value="0"/>-->
+    </target>
+
+    <define name="RADIO_CONTROL_AUTO1"/>
+
+    <configure name="USE_BARO_BOARD" value="TRUE"/>
+    <configure name="BARO_PERIODIC_FREQUENCY" value="50"/>
+
+    <module name="ahrs" type="float_dcm"/>
+    <module name="ins" type="alt_float"/>
+    <module name="control"/>
+    <module name="navigation"/>
+    <module name="auto1_commands"/>
+
+  </firmware>
+
+  <!-- PWM Servos connected to the regular Flightcontroller PWM outputs -->
+<!--  <servos driver="Pwm">-->
+<!--    <servo name="CAMTILT" no="3" min="1100" neutral="1500" max="1900"/>-->
+<!--    <servo name="CAMPITCH" no="4"  min="1900" neutral="1500" max="1100"/>-->
+<!--  </servos>-->
+
+
+  <!-- This driver name should match to the actuator module name -->
+  <!-- cASe SeNsITiVe! -->
+  <servos driver="T4">
+    <!-- ref: https://github.com/tudelft/Teensy_actuators_PPZ -->
+    <!-- no="0~9" for servos, no="10~13" for escs -->
+    <!-- Servo no == (servo_ID - 1) because it starts from ZERO... -->
+    <!-- e.g. if you want to use a servo with ID 3 for your elevator -->
+    <!-- <servo name="ELEVATOR" no="2" min="20" neutral="50" max="100"/>-->
+
+    <servo name="S_THROTTLE" no="10" min="100" neutral="100" max="1900"/>
+
+<!--    degress * 100 -->
+    <servo name="S_AILERON_L"  no="6" min="-18000" neutral="0" max="18000"/>
+    <servo name="S_ELEVATOR" no="7" min="-18000" neutral="0" max="18000"/>
+    <servo name="S_RUDDER"   no="8" min="-18000" neutral="0" max="18000"/>
+    <servo name="S_AILERON_R"   no="9" min="-18000" neutral="0" max="18000"/>
+    <servo name="S_1" no="0" min="-18000" neutral="0" max="18000"/>
+    <servo name="S_2" no="1" min="-18000" neutral="0" max="18000"/>
+    <servo name="S_3" no="2" min="-18000" neutral="0" max="18000"/>
+    <servo name="S_4" no="3" min="-18000" neutral="0" max="18000"/>
+    <servo name="S_5" no="4" min="-18000" neutral="0" max="18000"/>
+    <servo name="S_6" no="5" min="-18000" neutral="0" max="18000"/>
+
+    <!--    <servo name="S_ID5"   no="4" min="20" neutral="510" max="1000"/>-->
+<!--    <servo name="S_ID6"   no="5" min="1000" neutral="510" max="20"/>-->
+<!--    <servo name="S_ID7"   no="6" min="20" neutral="510" max="1000"/>-->
+<!--    <servo name="S_ID8"   no="7" min="20" neutral="510" max="1000"/>-->
+<!--    <servo name="S_ID9"   no="8" min="20" neutral="510" max="1000"/>-->
+<!--    <servo name="S_ID10"   no="9" min="20" neutral="510" max="1000"/>-->
+  </servos>
+
+ <!--<servos driver="t4">--><!-- At the moment only one T4 board is supported, but feel free to enhance -->
```

Comment:
If only one T4 board is supported, I think this commented section can be removed